public/manifest.json

```diff
@@ -49,4 +49,4 @@
     },
     "default_title": "Marina"
   }
-}
+}
```

Comment:
Shouldnt be v3? @louisinger

Reply:
as the new updater is a pretty huge refacto, I thought it would be better to validate and discuss it first and then migrate to v3 (in a another PR maybe?). 

PS: theorically, it's totally possible to write a v3 manifest and tell webpack to build both v3 & v2 (I'm thinking of firefox)